docs/sidebars.js

```diff
@@ -485,115 +485,182 @@ module.exports = {
+              },
+              items: require("./docs/apis/resources/user_service/sidebar.js"),
+            },
+            {
+              type: "category",
+              label: "Session Lifecycle (Beta)",
+              link: {
+                type: "generated-index",
+                title: "Session Service API (Beta)",
+                slug: "/apis/resources/session_service",
+                description:
+                  "This API is intended to manage sessions in a ZITADEL instance.\n" +
+                  "\n" +
+                  "This project is in beta state. It can AND will continue breaking until the services provide the same functionality as the current login.",
+              },
+              items: require("./docs/apis/resources/session_service/sidebar.js"),
+            },
+            {
+              type: "category",
+              label: "OIDC Lifecycle (Beta)",
+              link: {
+                type: "generated-index",
+                title: "OIDC Service API (Beta)",
+                slug: "/apis/resources/oidc_service",
+                description:
+                  "Get OIDC Auth Request details and create callback URLs.\n" +
+                  "\n" +
+                  "This project is in beta state. It can AND will continue breaking until the services provide the same functionality as the current login.",
+              },
+              items: require("./docs/apis/resources/oidc_service/sidebar.js"),
+            },
+            {
+              type: "category",
+              label: "Settings Lifecycle (Beta)",
+              link: {
+                type: "generated-index",
+                title: "Settings Service API (Beta)",
+                slug: "/apis/resources/settings_service",
+                description:
+                  "This API is intended to manage settings in a ZITADEL instance.\n" +
+                  "\n" +
+                  "This project is in beta state. It can AND will continue to break until the services provide the same functionality as the current login.",
+              },
+              items: require("./docs/apis/resources/settings_service/sidebar.js"),
+            },
+          ]
         },
         {
           type: "category",
-          label: "Settings Lifecycle (Beta)",
+          label: "Resource Based (V3)",
+          collapsed: false,
           link: {
             type: "generated-index",
-            title: "Settings Service API (Beta)",
-            slug: "/apis/resources/settings_service",
-            description:
-              "This API is intended to manage settings in a ZITADEL instance.\n" +
-              "\n" +
-              "This project is in beta state. It can AND will continue to break until the services provide the same functionality as the current login.",
+            title: "Resource Based APIs (V3)",
+            slug: "/apis/resources_v3/",
+            description: "Resource based API definitions",
```

Comment:
I think we should add a more descriptive description here.
What does resource based API (v3) mean?
When should I use this one?

Maybe we can even link to a more detailed page, where we describe when to use what.